### DISCUSSION TITLE
Upgrade BigQuery Simba Driver to latest version to 1.3.2

### DISCUSSION THREAD

**Discussion by ak-workit** [dbeaver/dbeaver#19356] [General]
Upgrade BigQuery Simba Driver to latest version to 1.3.2 in the Dbeaver OSS please.
The reason being new driver would allow to read JSON column data from bigquery table.

**Comment by E1izabeth** [comment]
Please, create issue with this request.

**Reply by ak-workit** [reply]
Done
https://github.com/dbeaver/dbeaver/issues/19370

**Comment by LonwoLonwo** [comment]
Hello @arvind-workit 

You can always do it manually:

![2023-03-14 18_27_20-DBeaver Ultimate 23 0 0 - SYSROLEGRANT](https://user-images.githubusercontent.com/45152336/225051697-79d21abd-932b-4908-bee6-fc1da22c39fe.png)
